extensions/blocks/image-compare/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import icon from './icon';
+import save from './save';
+
+export const name = 'image-compare';
+
+export const settings = {
+	title: __( 'Image Compare', 'jetpack' ),
+	description: __( 'Compare two images with a slider.', 'jetpack' ),
+
+	icon,
+
+	category: 'layout',
```

Comment:
This would need fallback support for jetpack category, example:

https://github.com/Automattic/jetpack/blob/21daeebcab1798413b86f85bae01f3cc9f53ad92/extensions/blocks/business-hours/index.js#L86

Reply:
Done. 👍